src/main/java/at/hannibal2/skyhanni/features/event/carnival/CarnivalQuickStart.kt

```diff
@@ -5,6 +5,7 @@ import at.hannibal2.skyhanni.data.Perk
 import at.hannibal2.skyhanni.events.EntityClickEvent
 import at.hannibal2.skyhanni.events.LorenzChatEvent
 import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
```

Comment:
please make sure you have optimize imports option on in intellij when you are committing code. Also not related to this pr but ensure you have reformat code on so that code you write follows our fomatting styles

![image](https://github.com/user-attachments/assets/5e174c86-c6cf-4a84-b833-cc1fbdf6dac8)